src/App.jsx

```diff
@@ -1,13 +1,172 @@
+                               }}
+                        />
+                    </label>
+                    <label htmlFor="zipCode">
+                        Postcode
+                        <input id="zipCode"
+                               type="text"
+                               name="zipCode"
+                               value={zipCode}
+                               onChange={(e) =>
+                                   setZipCode(e.target.value)}
+                        />
+                    </label>
+                    <label htmlFor="delivery">
+                        Bezorgfrequentie
+                    </label>
+                    <select id="delivery"
+                            name="delivery"
+                            value={delivery}
+                            onChange={(e) =>
+                                setDelivery(e.target.value)}
+                    >
+                        <option value="Weekly"
+                        >Iedere week
+                        </option>
+                        <option value="Every other week"
+                        >Om de week
+                        </option>
+                        <option value="Monthly"
+                        >Iedere maand
+                        </option>
+                    </select>
+                    <label htmlFor="radio">
+                        <input type="radio"
+                               id="radio"
+                               name="radio"
+                               value={radioButton}
+                               onChange={() =>
+                                   setRadioButton("Overdag")}
+                        >
+                        </input>
+                        Overdag
+                        <input type="radio"
+                               id="radio"
+                               name="radio"
+                               value={radioButton}
+                               onChange={() =>
+                                   setRadioButton("'s Avonds")}
+                        >
+                        </input>
+                        {/* eslint-disable-next-line react/no-unescaped-entities */}
+                        's Avonds
+                    </label>
+                    <label>
+                        Opmerking
+                    </label>
+                    <textarea rows="5" cols="33">
+                    </textarea>
+                    <label>
+                        <input type="checkbox"
+                               id="checkbox"
+                               name="checkbox"
+                               onChange={() =>
+                                   setTermsAndConditions(!termsAndConditions)}
+                        >
+                        </input>
+                        Ik ga akkoord met de voorwaarden
+                    </label>
+                    <Button label="Verzend" onClick={() => console.log("Formulier verzonden!")} />
```

Comment:
Je gebruikt hier keurig het Button component gebruikt, echter heb je in jouw component `type="button"` staan. Maar de button staat binnen een form element en had dus eigenlijk een "submit" button moeten zijn. Je had dit kunnen oplossen door van type ook een property te maken.